<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegPrint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/recursos/printer.png</normaloff>:/recursos/printer.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Datos</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Marca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="inMarca">
         <property name="toolTip">
          <string>Elige la marca de tu impresora</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 226, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Modelo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inModelo">
         <property name="toolTip">
          <string>Ingresa el modelo de la impresora</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 226, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Ingresa el modelo de la impresora </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Tipo de impresion:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="inColor">
         <property name="toolTip">
          <string>seleccionalo si tu impresora es a color</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 226, 255);</string>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="inBN">
         <property name="toolTip">
          <string>seleccionalo si tu impresora es a Blanco y Negro</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 226, 255);</string>
         </property>
         <property name="text">
          <string>Blanco y Negro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Contador</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="inContador">
         <property name="toolTip">
          <string>total de impresiones realizadas por la maquina</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 226, 255);</string>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Precio:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="outPrecio">
         <property name="toolTip">
          <string>costo de la impresora seleccionada</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 226, 255);</string>
         </property>
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdAgregar">
         <property name="toolTip">
          <string>dale click al boton para agregar la impresora al registro</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 226, 255);</string>
         </property>
         <property name="text">
          <string>Agregar al registro</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>impresoras registradas</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="outDetalle">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 226, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuacerca_de">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuacerca_de"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/save.png</normaloff>:/recursos/save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/recursos/problem.png</normaloff>:/recursos/problem.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
